common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerRequest.java

```diff
@@ -176,6 +177,10 @@
     @SerializedName(SerializedNames.MULTIPLE_CLOUDS_SUPPORTED)
     private boolean mMultipleCloudsSupported;
 
+    @NonNull
+    @SerializedName(SerializedNames.AUTHORIZATION_AGENT)
+    private String mAuthorizationAgent;
```

Comment:
If this scenario is COBO-specific, can we use extra query parameter instead? (so that we don't expose this).

Reply:
Dissenting opinion: I'd like to avoid using query params to smuggle arguments into the broker unless those arguments' audience is the STS (ESTS/B2C) -- separation of concerns

Reply:
yup, we are using `AuthorizationAgent` as intended to send `Browser` vs `Webview`